libklog/libklog.c

```diff
@@ -86,6 +93,16 @@ static void pumpthr(void *arg)
 				break;
 			}
 		}
+
+		/* Just stop here until dmesg on the console is disabled */
```

Comment:
```suggestion
		/* Just stop here while dmesg on the console is disabled */
```
(or until is enabled)